src/templates/blog-post.js

```diff
@@ -69,7 +69,7 @@ const BlogPostTemplate = ({
 
   return (
     <MDXProvider components={{ ...components, ...shortCodes }}>
-      <Layout location={location}>
+      <Layout location={location} slug="Blog">
```

Comment:
slugs are usually lowercase to be URL friendly 🤔  but since you're using this "slug" in URL there won't be an issue, but just for semantic coherence maybe just rename the prop to "title"